lib/db/mysql/schema.sql

```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS tokens (
   FOREIGN KEY (clientId) REFERENCES clients(id) ON DELETE CASCADE,
   userId BINARY(16) NOT NULL,
   INDEX tokens_user_id(userId),
-  email VARCHAR(256) NOT NULL,
+  email VARCHAR(256),
```

Comment:
I don't know about this intermediate state where some tokens can't fetch the email address...ISTM maybe we should go through with something like https://github.com/mozilla/fxa-auth-server/issues/1053 and then we can just drop this column entirely.

In fact, I wonder if we can deprecate having the email returned implicitly by /verify, and it's a bit of an abstraction violation anyway...